src/components/Cookie/Cookie.tsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+
+import { useState, useEffect, useRef } from "react";
+import { Button } from "@/components";
+import Cookies from "js-cookie";
+import Link from "next/link";
+
+const Cookie = () => {
+  const [showCookie, setShowCookie] = useState(false);
+  const [consent, setConsent] = useState(false);
```

Comment:
Will probs need to chuck this in a context at some point, but fine here for now I think